capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -34,6 +34,35 @@ export class SearchUserModal extends Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    
+    if (this.props.club && this.props.type === 'clubs') {
+      if (this.props.deletedMemberId && (this.props.deletedMemberId !== prevProps.deletedMemberId)) {
+
+        const deleteMemberIndex = this.indexOfMemberArray(this.state.addedUsers, this.props.deletedMemberId);
+
+        if (deleteMemberIndex >= 0) {
+          const addedUsers = [...this.state.addedUsers];
+          addedUsers.splice(deleteMemberIndex, 1);
+
+          this.setState({ addedUsers });
+        }
+      }
+    }
+  }
+  
+  indexOfMemberArray = (arr, provideMemberId) => {
+    let index = 0
+
+    for (const member of arr) {
+      if (member.id === provideMemberId) {
+        return index;
+      }
+      index += 1;
+    }
+    return -1
+  }
```

Comment:
Good to make these sorts of things separate functions like you're doing here, but I think it can be removed if you use `filter` (or possibly `find`?) above

Reply:
Ok